valohai.yaml

```
@@ -0,0 +1,46 @@
+- step:
+    name: redshift-query
+    image: python:3.11-slim
+    command:
+    - pip install -r /valohai/repository/connectors/steps/redshift/requirements.txt -q
+    - PYTHONPATH=/valohai/repository python /valohai/repository/connectors/steps/redshift/query.py {parameters}
```

* `/valohai/repository` is the working directory, no need to add that here.
* Run the module with `python -m` to have Python set up `PYTHONPATH` internally correctly.
* We don't necessarily need `{parameters}` since we're using `valohai-utils`.

```suggestion
    - pip install -r connectors/steps/redshift/requirements.txt -q
    - python -m connectors.steps.redshift.query
```